samples/nrf9160/sms/prj.conf

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+# General config
+
+CONFIG_SMS_SEND_PHONE_NUMBER=""
+
+CONFIG_SMS=y
+CONFIG_SMS_SUBSCRIBERS_MAX_CNT=2
+CONFIG_SMS_SEND_CONCATENATED_MSG_MAX_CNT=2
+CONFIG_SMS_LOG_LEVEL_DBG=n
```

Comment:
`n` is the default, so this is not necessary.

Reply:
Yep, like CONFIG_SMS_SEND_PHONE_NUMBER, this is here for the convenience of the developer who will play with the sample or copy it as basis for his/her own app.